ui/src/app/service/data.service.ts

```diff
@@ -2,15 +2,15 @@ import {Injectable} from '@angular/core';
 
 import {Observable, Subject} from 'rxjs';
 
-import {TagpostServiceClient} from 'compiled_proto/src/proto/Tagpost_rpcServiceClientPb';
+import {Tag, Thread} from '../../compiled_proto/src/proto/tagpost_pb';
```

Comment:
Suggest to use absolute path at import stage.

Reply:
Changed to use absolute path in import. I will also fix this for other PRs sent.